src/classes/serializablePairPotential.cpp

```diff
@@ -57,6 +59,8 @@ void SerializablePairPotential::deserialise(const SerialisedValue &node)
     range_ = toml::find_or<double>(node, "range", 15.0);
     delta_ = toml::find_or<double>(node, "delta", 0.005);
     atomTypeChargeSource_ = toml::find_or<bool>(node, "includeCoulomb", false);
+    forceCharge_ = toml::find_or<bool>(node, "forceChargeSource", false);
+    autoCharge_ = toml::find_or<bool>(node, "autoChargeSource", false);
```

Comment:
The default value here should be `true` in order to be consistent with what we do in, for example, `Dissolve::clear()`.
```suggestion
    autoCharge_ = toml::find_or<bool>(node, "autoChargeSource", true);
```